paddle/fluid/operators/fill_constant_op.cc

```diff
@@ -26,6 +26,16 @@ class FillConstantOp : public framework::OperatorWithKernel {
                       "Output(Out) of FillConstantOp should not be null.");
 
     auto& shape = ctx->Attrs().Get<std::vector<int64_t>>("shape");
+    if (!ctx->HasInput("ShapeTensor") && !ctx->HasInputs("ShapeTensorList")) {
+      for (size_t i = 0; i < shape.size(); ++i) {
+        PADDLE_ENFORCE_GE(
+            shape[i], 0,
+            platform::errors::InvalidArgument(
+                "Each value of attribute 'shape' is expected to be greater "
+                "than 0. But recieved: shape[%u] = %d; shape = [%s].",
```

Comment:
It should be "greater  than or equal to 0". Maybe modify it next PR and I will approve this PR.

Reply:
OK. Thanks~ I will do in next PR.